crates/sqlexec/src/planner/logical_plan/drop_tables.rs

```diff
@@ -1,7 +1,7 @@
 use super::*;
 #[derive(Clone, Debug, Hash, PartialEq, Eq)]
 pub struct DropTables {
-    pub names: Vec<OwnedTableReference>,
+    pub references: Vec<OwnedFullObjectReference>,
```

Comment:
IMO `references` is a little vague. can we have this be `tbl_references`?

Reply:
I'm good with renaming.

I made this change to enforce that the execs are fully planned and there's no ambiguity when running them remotely.

As a side note, we should look at making changes to metastore to execute these DDL plans directly instead of converting them to metastore specific messages.

Reply:
> As a side note, we should look at making changes to metastore to execute these DDL plans directly instead of converting them to metastore specific messages.

i like that idea. Now that the execs are fully serializable, it seems unnecessary to convert them to yet another struct just so metastore can handle them.

Reply:
> IMO `references` is a little vague. can we have this be `tbl_references`?

Addressing this in https://github.com/GlareDB/glaredb/pull/1650